etl/steps/data/garden/tuberculosis/2023-11-27/burden_estimates.meta.yml

```diff
@@ -0,0 +1,248 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Tuberculosis
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/dataset/
+tables:
+  burden_estimates:
+    # Learn more about the available fields:
+    # http://docs.owid.io/projects/etl/architecture/metadata/reference/indicator/
+    title: Global tuberculosis report
+    description:
+      WHO has published a global tuberculosis (TB) report every year since
+      1997. The report provides a comprehensive and up-to-date assessment of the TB
+      epidemic, and of progress in prevention, diagnosis and treatment of the disease
+      at global, regional and country levels.
+    variables:
+      e_inc_100k:
+        title: Estimated incidence of all forms of tuberculosis
+        description_short: The number of new cases of [tuberculosis](#dod:tuberculosis) per 100,000 people.
+        unit: cases per 100,000 people
+        display:
+          numDecimalPlaces: 1
+      e_inc_100k_lo:
+        title: Estimated incidence of all forms of tuberculosis (low bound)
+        description_short: The number of new cases of [tuberculosis](#dod:tuberculosis) per 100,000 people.
+        unit: cases per 100,000 people
+        display:
+          numDecimalPlaces: 1
+      e_inc_100k_hi:
+        title: Estimated incidence of all forms of tuberculosis (high bound)
+        description_short: The number of new cases of [tuberculosis](#dod:tuberculosis) per 100,000 people.
+        unit: cases per 100,000 people
+        display:
+          numDecimalPlaces: 1
+      e_inc_num:
+        title: Estimated number of new cases of all forms of tuberculosis
+        unit: cases
+        display:
+          numDecimalPlaces: 0
+      e_inc_num_lo:
+        title: Estimated number of new cases of all forms of tuberculosis (low bound)
+        unit: cases
+        display:
+          numDecimalPlaces: 0
+      e_inc_num_hi:
+        title: Estimated number of new cases of all forms of tuberculosis (high bound)
+        unit: cases
+        display:
+          numDecimalPlaces: 0
+      e_tbhiv_prct:
+        title: Estimated HIV in incident tuberculosis
```

Comment:
I don't know if you will end up using these variables but a short_description would be useful here :) again I relate to the pain of metadata editing so it's all up to you as you know better which variables you will or won't end up using!